### DISCUSSION TITLE
do fury only on full releases, not prereleases (github actions)

### DISCUSSION THREAD

**Discussion by rfay** [goreleaser/goreleaser#3280] [Ideas, Issue Triage, and General discussions]
fury.io makes a mess of prereleases (v1.19.0-rc1) so I'd like to skip them entirely and only do major.minor.patch versions for fury.

The only ways I know to do this are to empty furies.account or empty FURY_TOKEN.

Since I have furies.account set to ${{ FURY_ACCOUNT }} I've tried something like this in github actions:

`FURY_ACCOUNT: {{ if not (contains "-" .github.ref) }}{{.secrets.FURY_ACCOUNT}}{{end}}`

But apparently github actions doesn't have "contains" as a function. 

Suggestions on how to keep semver prerelease tags with a "-" in them from publishing to fury.io?

Thanks!

**Comment by caarlos0** [comment]
I can't think of anything... maybe worth adding a `skip` like we do with publishers et al...

**Reply by rfay** [reply]
I think that would be a step forward, but I just tried another prerelease on fury using goreleaser and it seems to have done the right thing. So maybe they fixed it, or maybe I was wrong about the problem. Still allowing a skip would be a useful option. In general, DDEV would want a separate channel for prereleases, and that doesn't seem easy to do right now with fury and goreleaser. We do it for homebrew and AUR though.

**Reply by caarlos0** [reply]
will be in the next pro release :)
